src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -240,6 +224,26 @@ const ContactDetails = ({ systemIntake }: ContactDetailsProps) => {
     });
   };
 
+  /** Update contacts and system intake form */
+  const submit = async (callback?: () => void, validate?: boolean) => {
+    if (!isDirty) return callback?.();
+
+    // Update intake
+    const result = await updateSystemIntake();
+
+    if (!result?.errors) return callback?.();
+
+    // If validating form, show error on server error
+    if (validate) {
+      return setError('root', {
+        message: t('error:encounteredIssueTryAgain')
+      });
+    }
+
+    // If skipping errors, return callback
+    return callback?.();
+  };
+
```

Comment:
nit - can we use a default function to avoid the `?.()` chain?

something like `callback: () => void = () => {}` or `callback = () => {}`